sandpiper/processes/wps_evaluate_rule.py

```diff
@@ -0,0 +1,123 @@
+from pywps import Process, LiteralInput, LiteralOutput
+from pywps.app.Common import Metadata
+from functools import partial
+import json
+
+from p2a_impacts.fetch_data import get_dict_val
+from p2a_impacts.evaluator import evaluate_rule
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from sandpiper.utils import logger
+
+
+class EvaluateRule(Process):
+    """Evaluates the truth value of a climatological impact rule"""
+
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+
+        inputs = [
+            LiteralInput(
+                "rule", "Rule", abstract="Rule expression", data_type="string",
+            ),
+            LiteralInput(
+                "parse_tree",
+                "Parse Tree Dictionary",
+                abstract="File path to dictionary used for rule getter function",
+                data_type="string",
+            ),
+            LiteralInput(
+                "variables",
+                "Variable Dictionary",
+                abstract="File path to dictionary used for variables",
+                data_type="string",
+            ),
```

Comment:
We no longer want to pass filepath inputs as a LiteralInput since they won't be uploaded to the Docker Container. Try `supported_formats=[Format(mime_type="application/json", extension=".json",)],` and retrieve their filepath with `request.inputs[ComplexInput][0].file`

Reply:
Actually, ComplexInput [JSON(https://pywps.readthedocs.io/en/latest/_modules/pywps/inout/formats.html) is predefined in `pywps`, so use that instead for `supported_formats`.

Reply:
I tried this first and kept getting "file not found" when I tried to access the input. @sum1lim have you ever had this problem with JSON?

Reply:
@sum1lim I suggested to @cairosanders that she use a direct filepath as a workaround for the issue she was facing. I am unsure as to why this issue is occurring or how to fix it, so I suggest we create a separate issue for it and tackle it there.